common/file/file.go

```diff
@@ -58,3 +61,33 @@ func Exists(name string) bool {
 	_, err := os.Stat(name)
 	return !os.IsNotExist(err)
 }
+
+// WriteAsCSV dumps file as CSV
```

Comment:
Totally minor, but I think this could be changed to `WriteAsCSV takes a table of records and writes it as CSV`/something more descriptive since its not taking in a file to dump

Reply:
Yeah good suggestion just pushed change.